### ISSUE TITLE
5xx: Server Error

### ISSUE THREAD

**Issue by KStasi** [Codewars/codewars.com#1503]
Face 5xx: Server Error for the whole day. 
Sometimes I can reach home page but can't open any kata. I had this error few times before but after reloading it usually was solved. 
Could you help me? 

![image](https://user-images.githubusercontent.com/31795791/42227056-46cd1c3c-7ee9-11e8-9ae8-9d8267aa51ef.png)

**Comment by hobovsky** [comment]
And you did not read https://github.com/Codewars/codewars.com/issues/1502 by any chance, did you? :)

**Comment by kazk** [comment]
Closing as duplicate of #1502
